pkg/crc/preflight/preflight_check_tray_darwin.go

```diff
@@ -42,6 +42,7 @@ func checkIfDaemonPlistFileExists() error {
 }
 
 func fixDaemonPlistFileExists() error {
```

Comment:
Commit log is "Issue #1140 Load plist file before trying to start a launchd agent", feel free to detail why this is needed, why this was working before without this, ...